sdk/cosmos/azure-cosmos-spark-fabric-account-data-resolver/pom.xml

```diff
@@ -0,0 +1,848 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure.cosmos.spark</groupId>
+  <artifactId>azure-cosmos-spark-fabric-account-data-resolver</artifactId>
+  <version>1.0.0-beta.1</version>  <!-- {x-version-update;com.azure.cosmos.spark:azure-cosmos-spark-fabric-account-data-resolver;current} -->
+  <packaging>jar</packaging>
+  <name>OLTP Spark Connector for Azure Cosmos DB SQL API account data resolver for fabric</name>
+  <description>OLTP Spark Connector for Azure Cosmos DB SQL API account data resolver for fabric</description>
+  <properties>
+    <maven.build.timestamp.format>MM-dd-HH-mm-ss</maven.build.timestamp.format>
+    <jacoco.min.branchcoverage>0.17</jacoco.min.branchcoverage>
+    <jacoco.min.linecoverage>0.18</jacoco.min.linecoverage>
+    <maven.compiler.source>8</maven.compiler.source>
+    <maven.compiler.target>8</maven.compiler.target>
+    <jacoco.skip>true</jacoco.skip>
+    <shadingPrefix>azure_cosmos_spark_fabric</shadingPrefix>
+    <legal>
+      <![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]>
+    </legal>
+    <codesnippet.skip>true</codesnippet.skip>
+    <revapi.skip>true</revapi.skip>
+    <checkstyle.skip>true</checkstyle.skip>
+    <spotbugs.skip>true</spotbugs.skip>
+    <spotless.skip>true</spotless.skip>
+
+    <cosmos.spark.skip>false</cosmos.spark.skip>
+    <cosmos-spark-version>3.5</cosmos-spark-version>
+    <maven.main.skip>${cosmos.spark.skip}</maven.main.skip>
+    <maven.test.skip>${cosmos.spark.skip}</maven.test.skip>
+
+    <cosmos-spark-version>3.5</cosmos-spark-version>
+  </properties>
+
+  <developers>
+    <developer>
+      <id>microsoft</id>
+      <name>Microsoft</name>
+    </developer>
+  </developers>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-sql_2.12</artifactId>
+      <version>3.5.0</version> <!-- {x-version-update;cosmos-spark_3-5_org.apache.spark:spark-sql_2.12;external_dependency} -->
+      <exclusions>
+        <exclusion>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-all</artifactId>
```

Comment:
please check the recent exclusions for slf4j etc. I had to add - I expect this might be needed here as well.